calculator/src/components/GrowthInput.js

```diff
@@ -0,0 +1,85 @@
+import React,{useState} from "react";
+import {Bar} from 'react-chartjs-2';
+import {Form} from 'react-bootstrap'
+
+export default function GrowthInput(props) {
```

Comment:
since you're not sending props you could not write it because you won't be using them

Reply:
sorry i copy pasted the from my other project to get the structure and forgot to remove props.